gallery/src/components/NxButton/NxButtonPage.tsx

```diff
@@ -59,7 +59,9 @@ export default function NxButtonPage() {
                 A string to render as a tooltip and accessibility label for the button. This is generally not necessary
                 for buttons that include text content, but icon-only buttons should use this to make the button's
                 meaning clear in all contexts. Omitting this prop when using an icon-only button is deprecated and will
-                become unsupported in a future release.
+                become unsupported in a future release. If <NxCode>NxButton</NxCode> contains both
+                {' '}<NxCode>title</NxCode> and <NxCode>disabled</NxCode> props, <NxCode>NxButton</NxCode> will throw an
```

Comment:
I wonder if it is a good idea to wrap this inside `NxWarningAlert`

Reply:
> I wonder if it is a good idea to wrap this inside `NxWarningAlert`

Wrapped text in a warning in this commit: [5d26c0b](https://github.com/sonatype/sonatype-react-shared-components/pull/729/commits/5d26c0bcaeffd6f8334a903ec5fbf5a5d759d326)